docs/concepts/jcloud/configurations.md

```diff
@@ -33,119 +31,93 @@ Since each Executor has its own business logic, it may require different cloud r
 
 In JCloud, you can pass highly customizable, finely-grained resource requests for each Executor using the `jcloud.resources` argument in your Flow YAML.
 
+### Instance
 
-### CPU
+JCloud uses the concept of an "instance" to represent a specific set of hardware specifications.
+In above example, a C4 instance type represents two cores and 4 GB RAM based on the CPU Tiers instance definition table below.
 
-By default, `0.1 (1/10 of a core)` CPU is allocated to each Executor. You can use the `cpu` argument under `resources` to change it.
-
-JCloud offers the general Intel Xeon processor (Skylake 8175M or Cascade Lake 8259CL) by default. 
-
-```{hint}
-Maximum of 16 cores is allowed per Executor.
-```
+Note that if you are still using the legacy resource specification interface, such as:
```

Comment:
```suggestion
Note that we will translate the raw numbers from input to instance tier that fits most closely if you are still using the legacy resource specification interface, such as:
```

Suggest wrapping this in an admonition. I'd add the code but since it includes lots of backticks GH won't render it. Just `grep admonition` in the `docs/` folder and you'll see some examples

Reply:
ok fixed